.github/workflows/python_wheels.yml

```diff
@@ -193,8 +193,6 @@ jobs:
             pip_install_flags: '--user'
           - os_image: ubuntu:24.04
             pip_install_flags: '--user'
-          - os_image: fedora:39
-            pip_install_flags: ''
```

Comment:
perhaps add FIXME to reenable fedora and opensuse?

Reply:
fedora is reenabled - maybe you meant ubuntu?
Sure.